system/lib/wasmfs/js_api.cpp

```diff
@@ -22,11 +22,15 @@ using namespace wasmfs;
 
 extern "C" {
 
+// TODO: Replace forward declarations with #include <emscripten/wasmfs.h> and resolve wasmfs::backend_t namespace conflicts.
```

Comment:
It looks like this comment extends beyond 80 chars. It would be good to run `git clang-format main` (might require installing clang-format) to make sure the code formatting is consistent.